docs/discover/field-filter.asciidoc

```diff
@@ -40,6 +40,31 @@ field name. This excludes documents that contain that value in the field.
 *Exists* button image:images/ExistsButton.jpg[Exists Button] to the right of the
 field name. This includes only those documents that contain the field.
 
+To manually add a filter:
+
+. Click *Add Filter*. A popup will be displayed for you to create the filter.
++
+image::images/add_filter.png[]
+. Choose a field to filter by. This list of fields will include fields from the
+index pattern you are currently querying against.
++
+image::images/add_filter_field.png[]
+. Choose an operation for your filter. You may filter where a field matches a
+particular value by selecting *is*, where a filter doesn't match a particular
+value by selecting *is not*, where a field is present (*exists*), or where a
+value for a field is within a range (*is between*). In addition, you can filter
+where a field *is one of* a particular list of values.
```

Comment:
Would this list be better represented as a table? Explicitly describing exactly how each operator works. It might be overkill, but at least we're not missing anything. There are 6 different operators. I added something similar [here](https://www.elastic.co/guide/en/kibana/current/timelion-conditional.html) for the options available in Timelion